LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/border-image-space-001.html

```diff
@@ -6,7 +6,7 @@
   <link rel="author" title="Levi Weintraub" href="mailto:leviw@chromium.org">
   <link rel="help" href="http://www.w3.org/TR/css3-background/#the-border-image-repeat">
   <meta name="assert" content="border-image-repeat: space property spaces out background image that doesn't fit an even number of times.">
-  <meta name="fuzzy" content="maxDifference=0-255; totalPixels=0-3895">
+  <meta name="fuzzy" content="maxDifference=0-80; totalPixels=0-1728">
```

Comment:
This is fine, but one of the two values might need some slight increasing depending on what EWS gives.

Reply:
Yes -- I was going to wait to see what EWS says. It's likely I'll need to do something to account for the case Bri modified with the very wide range of values.